### PR TITLE
Feature/extend functionality

### PR DESCRIPTION
**Pull request by fpodshivadlov**
Switched from node-fetch to support commonjs and the latest version of node;
Added VS code debug config;
Added fallback meaning finding;
Made word list not required parameter;
Added filter by topics;
Switched to linkedom due to potential memory leaks because of jsdom;
Added both American and British English support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,28 +2,30 @@
...
-  },
+  "keywords": [
+    "anki",
+    "englishprofile"
+  ],
   "author": "",
   "license": "ISC",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "ts-node ./src/index.ts",
```
Comment (alexander-rykhlitskiy) [inline package.json:13]: Thanks for your amazing contribution! It's really valuable for our project!

Why have you decided to use `start`? Isn't `run` more common for scripts?

Comment (fpodshivadlov) [reply package.json:13]: I thought [`npm start`](https://docs.npmjs.com/cli/v8/commands/npm-start) is common for all the npm packages

Comment (alexander-rykhlitskiy) [reply package.json:13]: Tests, build and other scripts are run with [npm run](https://docs.npmjs.com/cli/v8/commands/npm-run-script).

Comment (fpodshivadlov) [reply package.json:13]: You can use `npm run start` as well :)

**Pull request merged**